src/sentry/api/endpoints/organization_events_timeseries.py

```diff
@@ -377,3 +395,9 @@ def serialize_timeseries(self, result: SnubaTSResult, axis: str, rollup: int) ->
         timeseries["values"] = timeseries_values
 
         return timeseries
+
+    def check_incomplete(self, now: float, row: dict[str, Any]) -> str | None:
+        if now - row["time"] < INGESTION_DELAY:
+            return INGESTION_DELAY_MESSAGE
+        else:
+            return None
```

Comment:
👀 this is a nit, but when we do this on the frontend, we also take the bucket size into account

e.g., if if the current time is 1000, the current row time is 900, the bucket size is 100, and the delay is 90, we mark the last bucket as incomplete because the current time in within the bucket. Would be nice to have that here, too!

Reply:
copied the code from FE 👍